templates/gatsby-wordpress/files/.platform/routes.yaml

```diff
@@ -0,0 +1,16 @@
+"https://www.{default}/":
+  type: upstream
+  upstream: "gatsby:http"
+
+"https://{default}/":
+    type: redirect
+    to: "https://www.{default}/"
+
+"https://www.backend.{default}/":
+  type: upstream
+  upstream: "wordpress:http"
+
```

Comment:
```suggestion
```

The backend doesn't need a www, I'd think.

Reply:
For here I might. I'm using the `id` in the redirect (which in my testing didn't look like the upstream can have), and piping the redirects `to` into Gatsby to get all of the content. So for this setup I think I still need an upstream defined separately.